app/assets/stylesheets/style.scss

```diff
@@ -841,14 +842,14 @@ label:hover {
 .index-about {
   padding-top: 30px;
   p {
-   white-space: pre;
-   wrap: break-word;
-   color: black;
-   padding-left: 5px;
- }
- a {
-  display: inline-block;
-}
+    white-space: pre;
+    wrap: break-word;
+    color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.